lib/index.js

```diff
@@ -66,15 +75,31 @@ export const bbReducer = (backboneEntities) => () => backboneEntities.toJSON();
  *                                third-party capabilities such as middleware, time travel, persistence, etc.
  * @returns {Object} Object that implements a Redux-like store API.
  */
-export const createStore = (reducer, initialState, enhancer) => {
+export const createStore = (reducer, initialState = {}, enhancer) => {
```

Comment:
See note in description about not having updated the documentation yet. :P

On Fri, Aug 19, 2016 at 4:59 PM, Vitor Balocco <notifications@github.com>
wrote:

> In lib/index.js
> <https://github.com/adamterlson/backbone-redux-store/pull/6#discussion_r75557656>
> :
>
> > @@ -66,15 +75,31 @@ export const bbReducer = (backboneEntities) => () => backboneEntities.toJSON();
> >   *                                third-party capabilities such as middleware, time travel, persistence, etc.
> >   * @returns {Object} Object that implements a Redux-like store API.
> >   */
> > -export const createStore = (reducer, initialState, enhancer) => {
> > +export const createStore = (reducer, initialState = {}, enhancer) => {
>
> the jsdoc for initialState needs to be updated as well!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/adamterlson/backbone-redux-store/pull/6/files/6405c1fd44311b72ff41daabea06bba4de6664a2#r75557656>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACYfctIVBFRHc_yZFerk4NFF95GWyKlTks5qhidcgaJpZM4Jo4YV>
> .
>